### PR TITLE
BIPT-32

### PR DESCRIPTION
**Pull request by KaceyWood8**
### Related Ticket

[<[https://eyblockchain.atlassian.net/browse/__TICKET__>](https://eyblockchain.atlassian.net/browse/__TICKET__%3E)](https://eyblockchain.atlassian.net/browse/BIPT-32)

### Description

Once clicked on ‘New Proposal’, the user should be navigated to the New Proposal page with the additional required fields:

- BIP # (auto generated)
- Title of BIP
- Author
- Date Proposed (auto generated)
- Date Ratified
- Championship Team (volunteers helping support) (optional)
- Leadership Sponsor (dropdown with partner/SM options)
- Simple Summary/Abstract
- Motivation(s)
- Specifications
- Risks/Impediments
- Success Metrics
- Status (Draft, RFC, Voting, Accepted, Rejected)

image

### QA Instructions
- [ ] Step 1: git clone https://github.com/KaceyWood8/eybc-bip.git
- [ ] Step 2: npm i
- [ ] Step 3: View updated files: "src/components/proposalForm.tsx", and "node/modules/.prisma/client/index.d.ts"
- [ ] Step 4: npm run dev
- [ ] Step 5: Open http://localhost:3000/ with your browser to see the result.
- [ ] Step 6: Connect Metamask wallet to log in
- [ ] Step 7: Click on "New Proposal" in top right hand corner of screen.

### CODE REVIEW

**Comment by meaghanfitzgerald** [comment]
Why are `Championship Team` and `Date Proposed` not required fields?

**Comment by KaceyWood8** [comment]
@meaghanfitzgerald In BIPT-3 ticket it shows Championship Team in the bullets as optional. For Date Proposed, I assumed it would be optional in case someone wants to save it as draft (Also assumed once it is autogenerated it wouldn't need it to be required since the current date would be filled. No action required from the user for that field)

File: components/proposalForm.tsx [hunk 1]
```diff
@@ -33,72 +34,109 @@ const ProposalForm = () => {
...
+            <option value="option1">Chen Zur</option>
+            <option value="option2">James Canterbury</option>
+            <option value="option3">Rajat Kapur</option>
+            <option value="option4">Arwin Holmes</option>
+            <option value="option5">AJ May</option>
+            <option value="option6">Graham Cathcart</option>
+            <option value="option7">Brian Stern</option>
+            <option value="option8">Tom Garlick</option>
+            <option value="option9">Dale Bocra</option>
+            <option value="option10">Kartheek Solipuram</option>
```
Comment (nagrawal3) [inline components/proposalForm.tsx:88]: use this as an environment variable. Reason: if in the future we have to modify the list, we will not need to make changes to the code, only the environment variable needed to be changed.

File: example.env.local [hunk 2]
```diff
@@ -1,6 +0,0 @@
-# Copy this file and rename to .env.local
```
Comment (nagrawal3) [inline example.env.local:1]: leave this file here, do not delete

**Review by nagrawal3 — CHANGES_REQUESTED** [review]

File: components/proposalForm.tsx (hunk 1 above)
Comment (KaceyWood8) [reply components/proposalForm.tsx]: Hi @nagrawal3 I have added this to my latest commit. You can find it in fixtures/practiceleaders/tsx. Please let me know if there is a better way of doing this. Happy to connect online for any updates!

File: example.env.local (hunk 2 above)
Comment (KaceyWood8) [reply example.env.local]: @nagrawal3 Added file back in latest commit.

**Review by nagrawal3 — APPROVED** [review]

**Pull request merged**